aidial_adapter_bedrock/llm/converse/input.py

```diff
@@ -31,6 +36,25 @@
 from aidial_adapter_bedrock.llm.errors import ValidationError
 from aidial_adapter_bedrock.utils.list import group_by
 from aidial_adapter_bedrock.utils.list_projection import ListProjection
+from aidial_adapter_bedrock.utils.resource import Resource
+
+IMAGE_MIME_TO_CONVERSE_TYPE = {
```

Comment:
It's the types supported by the Converse API in **general**.

Where are the supported files for **particular** models defined? 
The models vary in the support of vision and doc chat.

Reply:
Fixed, added `supported_image_types` and `supported_document_types` properties for `ConverseAdapter`, and for each existing model